geode-core/src/main/java/org/apache/geode/cache/asyncqueue/internal/AsyncEventQueueStats.java

```diff
@@ -42,6 +42,8 @@
             f.createLongCounter(EVENT_QUEUE_TIME, "Total time spent queueing events.",
                 "nanoseconds"),
             f.createIntGauge(EVENT_QUEUE_SIZE, "Size of the event queue.", "operations", false),
+            f.createIntGauge(EVENT_SECONDARY_QUEUE_SIZE, "Size of the event secondary queue.",
```

Comment:
Maybe "Size of the secondary event queue" or "Number of secondary events in queue"

Reply:
I just want to follow the same term with "Size of the event queue"